php-frontend/src/main/java/org/sonar/php/api/PHPPunctuator.java

```diff
@@ -72,6 +72,7 @@
   NULL_COALESCE("??"),
 
   ARROW("->"),
+  SAFE_ARROW("?->"),
```

Comment:
Maybe we can rename it to `NULL_SAFE_ARROW` to be more explicit and more consistent with `NULL_COALESCE` and `NULL_COALESCING_EQU`.